src/adcm_client/packer/naming_rules.py

```diff
@@ -67,7 +67,7 @@ def check_version(version):
             "-" in position {version.index("-")}').with_traceback(sys.exc_info()[2])
 
 
-def resolve_build_id(git_data, master_branches, timestamp):
+def resolve_build_id(git_data, master_branches, timestamp, timestamp_enabled):
```

Comment:
brakes backward compatibility for this function.

Reply:
`def resolve_build_id(git_data, master_branches, timestamp, *args, **kwargs)` as solution